src/XenGnttab.cpp

```diff
@@ -104,6 +104,8 @@ void XenGnttabBuffer::release()
 	}
 }
 
+#ifdef GNTDEV_DMA_FLAG_WC
```

Comment:
I would add a comment that this is actually a workaround to get the knowledge if DMA supported by gntdev or not as there is no specific CONFIG_ option for that.
And it can of course change in the future